dtm0/fop.h

```diff
@@ -37,12 +50,52 @@ extern const struct m0_rpc_item_ops dtm0_req_fop_rpc_item_ops;
 int m0_dtm0_fop_init(void);
 void m0_dtm0_fop_fini(void);
 
+/**
+ * DTM0S operation flags.
+ */
+enum m0_dtm0s_op_flags {
+	/**
+	 * Define any operation flags
+	 */
+	DOF_NONE = 0,
+	/**
+	 * Delay reply until transaction is persisted.
+	 */
+	DOF_SYNC_WAIT = 1 << 0,
```

Comment:
what's the usecase for this flag?

Reply:
This change is leftover from my earlier changes. It is not needed anymore. Will remove it. Thanks!